### PR TITLE
CU-23kvwh6 - Remove unused integrations

### PR DESCRIPTION
**Pull request by mdzieciuchowicz**

### CODE REVIEW

**Comment by pmakowski23** [comment]
Task linked: [CU-23kvwh6 Remove unused integrations](https://app.clickup.com/t/23kvwh6)

File: components/register-confirmation/register-confirmation.module.css [hunk 1]
```diff
@@ -140,12 +140,12 @@
   }
 }
 
-.ticket-visual-wrapper {
+.register-confirmation-visual-wrapper {
```
Comment (pmakowski23) [inline components/register-confirmation/register-confirmation.module.css:143]: This and some other CSS classes are unused and can be also removed 😉 Please leave only classes that are used in `.../register-confirmation.tsx`

**Review by pmakowski23 — CHANGES_REQUESTED** [review]

File: components/register-confirmation/register-confirmation.module.css (hunk 1 above)
Comment (mdzieciuchowicz) [reply components/register-confirmation/register-confirmation.module.css]: Fixed

**Review by pmakowski23 — APPROVED** [review]
Great work!

File: components/register-confirmation/register-confirmation.module.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  .register-confirmation-layout {
+    grid-template-columns: 1fr 1fr;
+  }
+  .hero {
+    font-size: var(--space-16x);
+    text-align: left;
+  }
+  .description {
+    text-align: left;
+    margin: 30px 0;
```
Comment (pmakowski23) [inline components/register-confirmation/register-confirmation.module.css:59]: Tutaj chyba jeszcze bardzije

**Review by pmakowski23 — APPROVED** [review]
Wszystko git

**Review by oskarkurczewski — APPROVED** [review]

**Pull request merged**